component/admin/models/product_detail.php

```diff
@@ -1122,10 +1127,6 @@ public function store($data)
 
         $dispatcher->trigger('onAfterProductSave', array(&$row, $isNew));
 
-        if ($data['task'] === 'save2copy') {
```

Comment:
if you move this code block, it will affect to plugins using trigger onAfterProductSave